### PR TITLE
[PI-577] Create "Message Set" Device Reference Data

### PR DESCRIPTION
**Pull request by jaklinger**

### CODE REVIEW

File: infrastructure/swagger/05_paths.yaml [hunk 1]
```diff
@@ -343,6 +343,49 @@ paths:
...
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                questionnaire_responses:
+                  type: object
+                  description: Questionnaire Responses for MHS Message Set questionnaire
+      responses:
+        responses:
```
Comment (jameslinnell) [inline infrastructure/swagger/05_paths.yaml:377]: Need to create a response like in the swagger ticket.

File: infrastructure/swagger/05_paths.yaml [hunk 2]
```diff
@@ -343,6 +343,49 @@ paths:
...
+          in: path
+          required: true
+          description: logical identifier
+          schema:
+            type: string
+        - *RequestHeaderVersion
+        - *RequestHeaderRequestId
+        - *RequestHeaderCorrelationId
+      requestBody:
+        required: true
```
Comment (jameslinnell) [inline infrastructure/swagger/05_paths.yaml:367]: requestBody can go in the seperate file

File: infrastructure/swagger/05_paths.yaml [hunk 3]
```diff
@@ -343,6 +343,49 @@ paths:
...
+            schema:
+              type: object
+              properties:
+                questionnaire_responses:
+                  type: object
+                  description: Questionnaire Responses for MHS Message Set questionnaire
+      responses:
+        responses:
+        "201":
+          <<: *Response201
```
Comment (jameslinnell) [inline infrastructure/swagger/05_paths.yaml:379]: These responses won't exist anymore

File: infrastructure/swagger/05_paths.yaml (hunk 1 above)
Comment (jaklinger) [reply infrastructure/swagger/05_paths.yaml:373]: i didn't want to conflict with the swagger ticket by making changes here, depending on what goes in first i'd say best to put in as a fix-forward?

Comment (jameslinnell) [reply infrastructure/swagger/05_paths.yaml:373]: Yeah, not sure what the best option is here. I guess you are right. I'll fix it in my swagger ticket

**Review by jameslinnell — APPROVED** [review]

**Pull request merged**